### PR TITLE
Added optimism goerli to blastapi

### PR DESCRIPTION
**Pull request by LeifuChen**
## Description
Added optimism goerli node to blast api

### CODE REVIEW

File: constants/network.ts [hunk 1]
```diff
@@ -19,10 +19,12 @@ export enum BlastNetwork {
 	ETHEREUM_MAINNET = 'eth-mainnet',
 	OPTIMISM_MAINNET = 'optimism-mainnet',
 	ETHEREUM_KOVAN = 'eth-kovan',
```
Comment (platschi) [inline constants/network.ts:21]: Suggest to switch to L1 Goerli here too and remove Kovan support once and for all

File: constants/links.ts [hunk 2]
```diff
@@ -43,7 +43,7 @@ export const EXTERNAL_LINKS = {
 	},
 	Explorer: {
 		Optimism: 'https://optimistic.etherscan.io/tx',
-		OptimismKovan: 'https://kovan-optimistic.etherscan.io/tx',
```
Comment (Tburm) [inline constants/links.ts:46]: Need to make sure we update this link in the table as well: https://github.com/Kwenta/kwenta/blob/c5e1e69b0bd41c22e6a3aed04ec9981155197020/sections/futures/TradingHistory/TradesHistoryTable.tsx#L117

**Review by Tburm — COMMENTED** [review]
Just one minor change requested

File: constants/links.ts (hunk 2 above)
Comment (LeifuChen) [reply constants/links.ts:46]: Thanks for the reminder. I made the change here. Can you see it?
https://github.com/Kwenta/kwenta/pull/1359/files#diff-67c44e4805a21ab8854d6af6609a9beeff1366b4668c1c8e86ce686910c112f7

File: sections/futures/TradingHistory/TradesHistoryTable.tsx [hunk 3]
```diff
@@ -114,7 +114,7 @@ const TradesHistoryTable: FC<TradesHistoryTableProps> = ({ numberOfTrades, mobil
 						row.original.id !== NO_VALUE
 							? isL2Mainnet
 								? window.open(`${EXTERNAL_LINKS.Explorer.Optimism}/${row.original.id}`)
-								: window.open(`${EXTERNAL_LINKS.Explorer.OptimismKovan}/${row.original.id}`)
+								: window.open(`${EXTERNAL_LINKS.Explorer.OptimismGoerli}/${row.original.id}`)
```
Comment (LeifuChen) [inline sections/futures/TradingHistory/TradesHistoryTable.tsx:117]: @Tburm here is the change.

Comment (Tburm) [reply sections/futures/TradingHistory/TradesHistoryTable.tsx:117]: Looks like there might be some other issue with this logic, when I am connected to OP Goerli and click a trade it routes to `optimistic.etherscan`

Comment (LeifuChen) [reply sections/futures/TradingHistory/TradesHistoryTable.tsx:117]: > Looks like there might be some other issue with this logic, when I am connected to OP Goerli and click a trade it routes to `optimistic.etherscan`

Thanks. I will investigate this issue.

Comment (LeifuChen) [reply sections/futures/TradingHistory/TradesHistoryTable.tsx:117]: @Tburm When I switched to OP Goerli and check the spot history table. There is no trade at all. I also checked the subgraph endpoint but didn't get any results. https://api.thegraph.com/subgraphs/name/kwenta/optimism-goerli-main/
image

**Review by avclarke — APPROVED** [review]
Changes look good, I think we can tackle the subgraph issue separately.

**Pull request merged**